kolkokrzyzykV2.py

```diff
@@ -0,0 +1,84 @@
+def restart():
+    answer = input('Do you want to play again?\nType "yes" or "no".\n')
+    if answer == 'yes':
+        tictactoe()
+    elif answer == 'no':
+        exit()
+    else:
+        print('Answer not recognized.\n')
+        restart()
+
+
+def tictactoe():
+    while True:
+        cells = '         '
+        who_moves = 1
+        cells_list = [x for x in cells]
+
+        def grid():
+            if who_moves % 2 == 1:
+                player = 'X'
+            else:
+                player = 'O'
+            print(f'\nNow player {player} moves.')
+            print("---------")
+            print('| ' + cells_list[0] + ' ' + cells_list[1] + ' ' + cells_list[2] + ' |')
+            print('| ' + cells_list[3] + ' ' + cells_list[4] + ' ' + cells_list[5] + ' |')
+            print('| ' + cells_list[6] + ' ' + cells_list[7] + ' ' + cells_list[8] + ' |')
+            print("---------")
+
+        def check():
+            row1 = cells_list[0:3]
+            row2 = cells_list[3:6]
+            row3 = cells_list[6:9]
+            column1 = [cells_list[0], cells_list[3], cells_list[6]]
+            column2 = [cells_list[1], cells_list[4], cells_list[7]]
+            column3 = [cells_list[2], cells_list[5], cells_list[8]]
+            diagonal1 = [cells_list[0], cells_list[4], cells_list[8]]
+            diagonal2 = [cells_list[2], cells_list[4], cells_list[6]]
+            win_con_x = ['X', 'X', 'X']
+            win_con_o = ['O', 'O', 'O']
+            matrix = [row1, row2, row3, column1, column2, column3, diagonal1, diagonal2]
+            if win_con_x in matrix:

```

Comment:
A co powiesz na sprawdzanie czy wartość w kolumnach, rzędach itp są takie same. Zamiast porównywać z wzorcem to sprawdzić czy `cells_list[0] == cells_list[1]  && cells_list[1] == cells_list[2]`